compiler/src/iree/compiler/Codegen/LLVMGPU/test/ROCDL/pipeline_tile_and_fuse.mlir

```diff
@@ -620,9 +648,11 @@ hal.executable public @main {
 
 #lowering_config = #iree_gpu.lowering_config<{
   mma_kind = #iree_gpu.mma_layout<WMMA_I32_16x16x16_I8>,
-  reduction = [0 : index, 0 : index, 4 : index],
-  subgroup = [2 : index, 4 : index, 0 : index],
-  workgroup = [64 : index, 128 : index, 0 : index]}>
+  reduction = [0, 0, 4],
+  subgroup = [2, 4, 0],
+  workgroup = [64, 128, 0],
+  promote_operands = [0, 1]
```

Comment:
As we add more fields to the lowering config other than tile sizes, I think it would be more clear to rename the tile size fileds to indicate that they are tile sizes (i.e., `reduction_tile_sizes`, etc.).